### ISSUE TITLE
2020-01-13 tc121 M3SD computation engine crashes with incorrect error that for mrt_ivif_p  DOF is non-numeric

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#152]
With commit dce4ed8

tc121 has been loaded to onedrive

```r
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
Error in value[[3L]](cond) : 
  Error in mrt_ivif_p(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : Error in mrt_ivif_p: dof is not a numeric value
For SDEID 49420991
In addition: Warning messages:
1: In validate_timeconc_data(map, data, flag, verbose = verbose) :
  No tau information provided in 'map'.
2: In validate_timeconc_data(map, data, flag, verbose = verbose) :
  No told information provided in 'map'.
3: In run_computation(data = d, map = mct, flag = flags, parameterset = parameterset) :
  'RETURNCOLS' values provided via 'map' are not used for this computation
4: In run_M3_SD_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in mrt_ivif_p(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : Error in mrt_ivif_p: dof is not a numeric value
For SDEID 49420991 
> class(d$DOF)
[1] "numeric"
```

**Comment by tensfeldt** [comment]
This impacts JIRA Issues EQ-2214 and EQ-2219
Note if all MRT and VSS related parameters are removed from PARAMETERLIST, the computation runs but of course omits required MRT and VSS parameters.

**Comment by rudraya** [comment]
For this issue I noticed that the MCT has the value for DOF under the name "DOF" for tc121. Currently the computation engine looks for "DOF1" for DOF. Adding the following line resolved this issue: 

```{r}
map_tc121$DOF1 <- "DOF"
```
Please let me know if you want change the name computation engine looks for or keep it the same as it is not, which is "DOF1"

**Comment by jhhughes256** [comment]
This also impacts tc123 which has `DOF` defined in the `mct` rather than `DOF1`. This issue has come up multiple times now (#85, #103, #108, #126). I believe this has been classifed as an `mct` issue in the past.

**Comment by tensfeldt** [comment]
This is NOT an MCT issue formally but a misimplementation of the computation engine specification for DOFi which clearly states: 

"Note that if there’s only one dose, DOF is a synonym for DOF1."

Since we've not been able to comply with the specification through the computation engine implementation, the only way to address it until the implementation is corrected and resolves this defect is to modify the associated MCTs temporarily to permit other required testing activities.

**Comment by rudraya** [comment]
This issue should be resolved as of b9858f9. Please let me know if it still occurs

**Comment by jhhughes256** [comment]
As of commit b9858f9 the computation engine accepts DOF as an alias for DOF1. As a result tc121 M3SD runs with successfully without changes to the MCT.
